### PR TITLE
Sync ff 0.13

### PR DESCRIPTION
**Pull request by zhenfeizhang**

### CODE REVIEW

**Comment by zhuoatscroll** [comment]
conflict

**Comment by zhenfeizhang** [comment]
> conflict

resolved

File: Cargo.toml [hunk 1]
```diff
@@ -27,16 +27,16 @@ debug-assertions = false
 lto = "fat" 
 # codegen-units = 1
 panic = "abort"
-incremental = true
+incremental = false
```
Comment (lispc) [inline Cargo.toml:30]: is this important?

**Comment by lispc** [comment]
oh the codes cannot even compile...

File: snark-verifier/src/system/halo2.rs [hunk 2]
```diff
@@ -13,6 +13,7 @@ use crate::{
     },
     util::{
         arithmetic::{root_of_unity, CurveAffine, Domain, FieldExt, Rotation},
+        arithmetic::{root_of_unity, CurveAffine, Domain, Rotation},
```
Comment (lispc) [inline snark-verifier/src/system/halo2.rs:16]: here..

**Pull request merged**